belgif-rest-problem/src/main/resources/io/github/belgif/rest/problem/Messages_fr.properties

```diff
@@ -0,0 +1,30 @@
+# problem detail messages
```

Comment:
@usku01 @jflabatBCSS could you pay some special attention to these French translations?